### DISCUSSION TITLE
Relating CaseArticle to Case

### DISCUSSION THREAD

**Discussion by mayday6971** [deejay-hub/timeline-lwc#219] [Q&A] [answered]
I also have a strange issue where I cannot link Case with CaseArticle. 

CaseArticle is linked to Case, but not like LinkedArticle is to say Incident.
https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_casearticle.htm

Has anyone had any luck with this configuration? It cascades down through the KnowledgeArticle.

Thanks in advance!

**Comment by deejay-hub** [comment]
<img width="1111" alt="image" src="https://user-images.githubusercontent.com/7021883/194444866-70f0cd3b-4ffb-42c4-8cc7-317d6c64a848.png">

See above for config example. You can use dot notation to traverse the CaseArticle -> KnowledgeArticle -> KnowledgeArticleVersion.  This positions based on the date it was attached to the case.

**Comment by deejay-hub** [comment, marked as answer]
In the example above I used the Fallback Tooltip Field to choose a field to display when the user hovers over a Case Article.

Another alternative option for the tooltip displayed when hovering is to use the compact layout for the Knowledge object (Setup->Object Manager->Knowledge). This in hindsight would be better since you can change the fields displayed declaratively. See the new config below.

<img width="1302" alt="image" src="https://user-images.githubusercontent.com/7021883/194462772-b8211970-823f-4315-84c9-9df5efaf0ed1.png">

Which looks like this to the user:

<img width="629" alt="image" src="https://user-images.githubusercontent.com/7021883/194474519-38d48d3a-a4d6-4fe7-99dc-cc512a009bd2.png">

**Comment by mayday6971** [comment]
I can confirm this worked perfectly. Thanks for the assistance!
